ozaria/site/common/ozariaUtils.js

```diff
@@ -8,15 +8,13 @@ export const defaultCodeLanguage = 'python'
  * Calculates all the next levels for a list of levels in a classroom/campaign based on the level sessions.
  * @param {Object[]} sessions - The list of level session objects.
  * @param {Object[]|Object} levels - The list of level objects, or an object with keys as level original id and value as level data.
- * @param {Object[]} levels.nextLevels - The array of nextLevels for a level.
- * @param {string} levels.nextLevels[].levelOriginal - Original id of the next level.
+ * @param {Object} levels.nextLevels - The array of nextLevels for a level.
+ * @param {Object} level.isPlayedInStages - True/false/undefined
```

Comment:
Should the type be the object parameter?
In this case:
```suggestion
 * @param {bool|undefined} level.isPlayedInStages - True/false/undefined
```

Reply:
Yep, good catch! I somehow forgot to change the type here, will update